packages/node_modules/@node-red/editor-client/src/sass/forms.scss

```diff
@@ -78,7 +78,8 @@
     }
 
     textarea {
```

Comment:
Apply this style change at the top level `textarea` will apply everywhere in the editor.
As this issue is limited to the ACE editor, I think this fix should be applied to a more specific CSS selector.

Inspected the DOM when typing in the editor, I see a `textarea` with class `"ace_text-input"` - is that the one we're seeing in this issue? If so, can we move this change to `ace.scss` and apply it to just `textarea.ace_text-input`?

Reply:
Thank you for reviewing it. I move the CSS definition to ace.scss. It also works correctly on Chrome, Firefox, Edge, and IE on Windows 10.